langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreIT.java

```diff
@@ -0,0 +1,57 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.client.MongoClient;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithFilteringIT;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.mongodb.MongoDBAtlasLocalContainer;
+
+import static dev.langchain4j.store.embedding.mongodb.TestHelper.*;
+
+public class MongoDbEmbeddingStoreIT extends EmbeddingStoreWithFilteringIT {
+
+    public static class ContainerIT extends MongoDbEmbeddingStoreWithRemovalIT {
```

Comment:
Would it be better to move this class to a separate file? I'm not sure if integration-test will be able to execute the class this way.

Reply:
Good catch on the extends, thanks!

I do not mind moving these to new files, will have a look.

Reply:
On the other hand, if we keep Cloud test and Local test in the same file, Could you please add `@EnabledIfEnvironmentVariable` in cloud tests to see whether `ContainerIT` will be triggered normally?

Reply:
Resolved, we are using just the testcontainer now, and this is done by checking the env vars in an if statement (since we can now also specify just a connection string).